skimage/transform/_seam_carving.pyx

```diff
@@ -0,0 +1,209 @@
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: nonecheck=False
+# cython: wraparound=False
+import numpy as np
+cimport numpy as cnp
+
+
+cdef cnp.double_t ABSOLUTE_MAX = np.finfo(np.double).max
```

Comment:
This is called `DBL_MAX` in various other parts of the code, e.g. `_corner_cy.pyx` and `_slic.pyx`. Rename for consistency.